src/components/layout/header/platform-switcher/PlatformCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { useTranslation } from 'react-i18next';
+import { TPlatformCard } from './Data';
+
+const PlatformCard = ({ src, title, alt, path }: TPlatformCard) => {
+    const { t } = useTranslation();
+
+    return (
+        <a href={path} className='mx-5 my-12 hover:cursor-pointer'>
+            <div className='flex min-h-[130px] flex-col gap-5 rounded-md p-5 hover:bg-disabled-100'>
+                <img className='w-36' src={src} alt={t(alt)} />
+                <span className='text-[14px] font-light'>{t(title)}</span>
```

Comment:
I'm not sure we can pass dynamic value to the translation or not we need to check this!
 
cc: @matin-deriv @ali-hosseini-deriv @balakrishna-deriv

Reply:
Later we may have an issue with the extractions

Reply:
@niloofar-deriv  it should work ,
is it possible to sort out this issue later ?
it wont be a big issue IMO

Reply:
all you have to do is do translation in Data.tsx instead of the component